src/trello2md.py

```diff
@@ -177,26 +184,32 @@ def main():
                     markdown.append(print_checklists(card['id'], data))
 
     # save result to file
-    if (args.output):
-        outputfile = args.output
+    to_stdout = False
+    if (args.to_stdout):
+      to_stdout = True 
     else:
-        outputfile = args.inputfile.replace('.json', '.md')
-        if outputfile == args.inputfile:
-            outputfile += '.md'
+      if (args.output):
+          outputfile = args.output
+      else:
+          outputfile = args.inputfile.replace('.json', '.md')
+          if outputfile == args.inputfile:
+              outputfile += '.md'
```

Comment:
What about

    to_stdout = args.to_stdout
    if not to_stdout:
      if args.output:
          outputfile = args.output
      else:
          outputfile = args.inputfile.replace('.json', '.md')
          if outputfile == args.inputfile:
              outputfile += '.md'